compiler/src/main/java/org/qbicc/graph/Throw.java

```diff
@@ -41,6 +41,10 @@ public Node getDependency() {
         return dependency;
     }
 
+    public boolean maySafePoint() {
+        return false;
```

Comment:
Does this imply that we won't be able to scan a Thread stack during exception delivery?  Probably ok, but maybe overly limiting?

Reply:
My thinking is that `Throw` is lowered to a native function which in turn doesn't safepoint. We can still scan the stack but we just can't rely on the immediate caller's frame for variable information. This should be OK though because all we need to do at this point is walk the stack return addresses which are generally stable even outside of safepoints (after all, we didn't even have safepoints until very recently 🙂).